extra_foam/utils.py

```diff
@@ -42,6 +42,31 @@ def timed_f(*args, **kwargs):
     return wrap
 
 
+
+class BlockTimer:
+    """A context manager for measuring the execution time of a code block
+
+    For example::
+
+        >>> with BlockTimer("foo"):
+        ...     time.sleep(1)
+        ...
+        Execution of foo: 1.001s
+    """
+    def __init__(self, label="block", enabled=True):
```

Comment:
You could add a quick description of the arguments, even if they are obvious

Reply:
True, added in 3ddcf067.